docs/ref/contrib/gis/functions.txt

```diff
@@ -0,0 +1,443 @@
+=============================
+Geographic Database Functions
+=============================
+
+.. module:: django.contrib.gis.db.models.functions
+    :synopsis: Geographic Database Functions
+
+.. versionadded:: 1.9
+
+The functions documented on this page allow users to access geographic database
+functions to be used in annotations, aggregations, or filters in Django.
+
+Example::
+
+    >>> from django.contrib.gis.db.models.functions import Length
+    >>> Track.objects.annotate(length=Length('line')).filter(length__gt=100)
+
+Not all backends support all functions, so refer to the documentation of each
+function to see if your database backend supports the function you want to use.
+If you call a geographic function on a backend that doesn't support it, you'll
+get a ``NotImplementedError`` exception.
+
+Function's summary:
+
+==================  =======================   ======================  ===================  ==================  =====================
+Measurement         Relationships             Operations              Editors              Output format       Miscellaneous
+==================  =======================   ======================  ===================  ==================  =====================
+:class:`Area`       :class:`BoundingCircle`   :class:`Difference`     :class:`ForceRHR`    :class:`AsGeoJSON`  :class:`MemSize`
+:class:`Distance`   :class:`Centroid`         :class:`Intersection`   :class:`Reverse`     :class:`AsGML`      :class:`NumGeometries`
+:class:`Length`     :class:`Envelope`         :class:`SymDifference`  :class:`Scale`       :class:`AsKML`      :class:`NumPoints`
+:class:`Perimeter`  :class:`PointOnSurface`   :class:`Union`          :class:`SnapToGrid`  :class:`AsSVG`
+
+                                                                      :class:`Transform`   :class:`GeoHash`
+
+                                                                      :class:`Translate`
+==================  =======================   ======================  ===================  ==================  =====================
+
+Area
+----
+
+.. class:: Area(expression, **extra)
+
+*Availability*: MySQL, Oracle, PostGIS, SpatiaLite
+
+Accepts a single geographic field or expression and returns the area of the
+field as an :class:`~django.contrib.gis.measure.Area` measure. On MySQL, a raw
+float value is returned, as it's not possible to automatically determine the
+unit of the field.
+
+AsGeoJSON
+---------
+
+.. class:: AsGeoJSON(expression, bbox=False, crs=False, precision=8, **extra)
+
+*Availability*: PostGIS, SpatiaLite
+
+Accepts a single geographic field or expression and returns a `GeoJSON`__
```

Comment:
Not sure if this was an intentional choice, but I prefer to inline URLs as it seems to make maintenance easier (won't insist on a change though).
```
`GeoJSON <http://geojson.org/>`__
```